### ISSUE TITLE
IP Camera Quality

### ISSUE THREAD

**Issue by Jemyy-AR** [AlexeyAB/darknet#2345]
Hello dear
I've trained my classifier model(darknet ) on my own dataset gathered from Open Image and Pascal(Cat, Dog, Others). 
my accuracy on my validation set is great( around 96%) but when I use my ip camera detected images to classify them I get terrible accuracy. I know my Ip camera's image doesn't have quality as much as my dataset but how can I improve my classification accuracy?
Is it possible to add some lag or noise to dataset at training time?
Or I have to increase the number of layers?

**Comment by aughey** [comment]
I've used IP cameras with good results.  Have you tried capturing still images from the camera and feeding it to darknet through the command line?  I would expect there to be comparable quality between the training data and the camera data.

**Comment by Jemyy-AR** [comment]
Dear @aughey 
Thanks for your response. Yes you're right. My dataset images at training time have higher quality than IP camera picture
I've added some gaussian noise to them for training, but accuracy is not still acceptable
Do you know any dataset for object detection or classification produced from IP camera's 
picture? Or a solution to my problem?

**Comment by GordonDongZHAO** [comment]
I think you can 
1. capture some IP camera images.
2. train with the captured images.

@Jemyy-AR
